modules/snooker/src/main/ts/ephox/snooker/resize/BarManager.ts

```diff
@@ -75,6 +77,10 @@ export const BarManager = (wire: ResizeWire): BarManager => {
     return newX - oldX;
   };
 
+  const setHoverRefresh = (state: boolean) => {
+    hoverRefresh = state;
+  };
```

Comment:
I noticed that it did TableResize -> BarManager -> Dragging and Dragging is part of dragster so I would have to expose some form of new isActive api. I was unsure if using that code path that I didn't use something that wasn't intended for. I'm not super familiar with snooker and dragster. I could make a stab at doing that change instead of introducing a new state since dragster has that state.

Reply:
Did the change seems to work fine. So good catch. I was thinking of using that first but was unsure what impact that would have.